src/navigation/services/navigation-entry.service.ts

```diff
@@ -107,6 +111,12 @@ export class NavigationEntryService {
       },
     });
 
+    await this.semanticProcessor.index(
```

Comment:
Could we wrap this on a trycatch? because Im getting this error on the index method?

![image](https://github.com/webtimemachine/wtm2/assets/88394108/acd6ab7b-2dc5-4980-a5e3-5661941c974f)

... and since the exception is not being handled the execution finish and the navigation entry is not being saved

Also I was wondering if we could add a section on the readme explaining how to get an openapi access token and also adding all the necesary instructions to run the LLM locally

Reply:
Done